src/utils/useDataChanges.ts

```diff
@@ -17,7 +19,8 @@ export const useDataChanges = () => {
   const dispatch = useDispatch();
   const setLanguage = (lang: string) => dispatch(actions.setLanguage(lang));
   const [, setDataChangeCount] = useGlobal('dataChangeCount');
-  const fetchUrl = useFetchUrlNow();
+  const ts: ISharedStrings = useSelector(sharedSelector, shallowEqual);
+  const fetchUrl = useFetchUrlNow(ts.expiredToken);
```

Comment:
If we didn't pass in the value but just expected it as a return value, we wouldn't need any changes to this module.